src/content/docs/usage/builders.mdx

```diff
@@ -6,83 +6,116 @@ sidebar:
+Daytona provides control over how the resulting Project environment is created, allowing you to choose your preferred method to build the Project image.
 
-When [creating a Workspace](/docs/usage/workspaces#create-a-workspace), you can choose between the following Builders during the Advanced Configuration step:
+You can choose between the following Builders while [creating a Workspace](/docs/usage/workspaces#create-a-workspace):
 
-1. [**Automatic**](#automatic-builder)
+- [**Automatic**](#automatic)
 
    Daytona builds the Project image after auto-detecting the most appropriate Builder.
 
-2. [**Dev Container**](#dev-container-builder)
+- [**Dev Container**](#dev-container)
 
    Daytona builds the Project image according to the Dev Container configuration located in the project.
 
-3. [**Custom Image**](#custom-image-builder)
+- [**Custom Image**](#custom-image)
 
    Daytona builds the Project image using an image you specify.
 
-4. [**None**](#none)
+- [**None**](#none)
 
    Daytona creates the Project using the default base image.
 
-## Automatic Builder
+## Automatic
 
-Daytona can auto-detect the most appropriate Builder for your project.
-When choosing this option, Daytona uses the following logic to determine which Builder to use:
+Daytona can auto-detect the most appropriate Builder for your Project. Daytona uses the following logic to determine which Builder to use:
 
-1. Check if a Dev Container configuration is present at `.devcontainer.json` or `.devcontainer/devcontainer.json`, relative to the repository root.
-2. If present, use the [Dev Container Builder](#dev-container-builder).
-3. If not present, use the [None Builder](#none).
+1. Check if a Dev Container configuration is present in **`.devcontainer.json`** or **`.devcontainer/devcontainer.json`**, relative to the repository root.
+2. If present, use the [Dev Container](#dev-container) Builder. If not present, use the [None](#none) Builder.
 
 To use the Automatic Builder, set the `--builder` flag value to `auto` during the [Workspace creation](/docs/usage/workspaces#create-a-workspace). This flag only applies when creating Workspaces with a single Project.
 
 ```shell
 daytona create <REPO_URL> --builder=auto
 ```
 
-## Dev Container Builder
+## Dev Container
+
+Development Containers (Dev Containers) provide a consistent and fully featured development environments for your [Project](/docs/usage/projects). A Dev Container defines all the necessary tools, runtime, and environment configurations required to build and run your Project.
+
+The Dev Container specification is designed to define a development environment that can be replicated across different platforms and CI/CD systems. The specification is defined within a `devcontainer.json` file in the project repository.
+
+The `devcontainer.json` file is a JSON file that consists of several components that define the development environment. The file includes the following properties:
```

Comment:
>  The file includes the following properties:

This seems like it's the only properties it has.

I think we should link to https://containers.dev/implementors/json_reference/ (I see we link to containers.dev below, but the comment still stands here)